spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -374,8 +392,10 @@ object DistriOptimizer {
         )
 
       } else {
-        logger.info(s"Warning!!! Ignore this iteration as more than maxDropPercentage " +
-          s"module is dropped!! Finished modules number: ${finishedModelNum}")
+        logger.info(s"Warning! Not enough training samples were successfully processed in this " +
+          s"iteration due to some slow tasks. The gradients computed in this iteration will be " +
+          s"discarded. Only $numFinishedModelUpdates/$driverSubModelNum threads successfully " +
```

Comment:
Personally I think showing both numbers is more informative. I can change it, though, if you feel strongly?